backend/src/schema/resolvers/statistics.js

```diff
@@ -28,6 +28,11 @@ export default {
           const stat = statistics[mapping[key]]
           response[key] = stat ? stat.toNumber() : 0
         })
+
+        /*
+         * Note: invites count is calculated this way because invitation codes are not in use yet
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-03T21:46:56Z" title="Thursday, October 3rd 2019, 11:46:56 pm +02:00">Oct 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -28,6 +28,11 @@ export default {
           const stat = statistics[mapping[key]]
           response[key] = stat ? stat.toNumber() : 0
         })
+
+        /*
+         * Note: invites count is calculated this way because invitation codes are not in use yet
```


We learned the day before yesterday that we open registration for non-donators in two weeks from now. So, Invitation codes will not be necessary anymore. (Just as a side note)

Reply:
> [<img alt="vbelolapotkov" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vbelolapotkov) **Authored by [vbelolapotkov](https://github.com/vbelolapotkov)**
_<time datetime="2019-10-04T08:14:57Z" title="Friday, October 4th 2019, 10:14:57 am +02:00">Oct 4, 2019</time>_

---


Thanks, good to know about invitation codes.